skimage/future/graph/graph_cut.py

```diff
@@ -75,7 +75,7 @@ def cut_threshold(labels, rag, thresh, in_place=True):
 
 
 def cut_normalized(labels, rag, thresh=0.001, num_cuts=10, in_place=True,
-                   max_edge=1.0):
+                   max_edge=1.0,  max_rec=float('Inf')):
```

Comment:
I have made it infinity to preserve the old behavior. I guess we can also change this to a big integer if it is an issue.